lib/utilities/markdown-color.js

```diff
@@ -177,7 +177,7 @@ function getToken(name, options) {
     name,
     token: `<${name}></${name}>`,
     pattern: getColorTokenRegex(name),
-    render: renderStylesFactory(renderer, name),
+    render: this.renderStylesFactory(renderer, name),
```

Comment:
Right, this was a tricky one. I'm binding `this`: https://github.com/ember-cli/ember-cli/pull/7259/files#diff-f6db1f1718f3ab9018f41d583b46f03dR111. Should I pull it into the class? I didn't want to change the surface API.